examples/bert/bert_cuda_gpu.json

```diff
@@ -50,9 +50,6 @@
         "transformers_optimization": {
             "type": "OrtTransformersOptimization",
             "config": {
-                "model_type": "bert",
-                "num_heads": 12,
-                "hidden_size": 768,
                 "float16": true
             }
```

Comment:
Presumably these are coming from the auto_tune_config  but I don't see it in the .json ?

Reply:
It will fetch the config from huggingface model config firstly, if it is not found, the use the settings from auto_tune_config.

The progress is be like:
1. fuse auto_tune_config with huggingface model config(if have)
2. output model will inherit the configs from first steps.
3. pass to parse the required configs.